### ISSUE TITLE
IndexError: pop from empty list + "diverges for 1 blocks"

### ISSUE THREAD

**Issue by KevinBatdorf** [skepticoin/skepticoin#110]
Tried running this and got this error message. Just curious what it means and how I can reconcile things

```
  diverges for 1 blocks

Traceback (most recent call last):
  File "/usr/local/bin/skepticoin-mine", line 8, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.9/site-packages/skepticoin/scripts/mine.py", line 6, in main
    miner_watcher()
  File "/usr/local/lib/python3.9/site-packages/skepticoin/mining.py", line 123, in __call__
    self.public_key = self.wallet.get_annotated_public_key("reserved for potentially mined block")
  File "/usr/local/lib/python3.9/site-packages/skepticoin/wallet.py", line 49, in get_annotated_public_key
    public_key = self.unused_public_keys.pop()
IndexError: pop from empty list
```

**Comment by atramos** [comment]
will be fixed in PR#113

**Comment by atramos** [comment]
Updated issue title... two different issues here, 

- the "IndexError: pop from empty list" is fixed in #112 / #113 
- "diverges for 1 blocks" occurs when there are no peers on the network, as the fix in #100 suppresses persisting mined blocks when disconnected
